test/libsolidity/syntaxTests/dataLocations/internalFunction/function_argument_location_specifier_test_internal_calldata.sol

```diff
@@ -2,4 +2,4 @@ contract test {
     function f(bytes calldata) internal;
 }
 // ----
-// TypeError: (31-36): Variable cannot be declared as "calldata" (remove the "calldata" keyword).
+// TypeError: (31-36): Storage location must be one of: "storage", "memory" for parameter in internal function, but "calldata" was given.
```

Comment:
I think it would sound better without `:`, at least with `:` what follows is not a real sentence.